python/cuspatial/cuspatial/tests/test_spline_fit.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import numpy as np
+import pytest
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+import cuspatial
+
+
+@pytest.mark.xfail(raises=TypeError, reason="size must be an integer")
```

Comment:
Instead of xfailing a test like this you should use `pytest.raises` and assert it raises the expected exception so it passes.

Reply:
👍